### PR TITLE
BUG: fixed artifact reloading to depend on directory

### PR DESCRIPTION
**Pull request by ebolyen**
Very minor changes to make artifacts consistent with working directory in multiple contexts (job, initial load, etc)

@Kleptobismol @jakereps can you review?

### CODE REVIEW

**Comment by jakereps** [comment]
It doesn't seem to be getting the created artifacts once it reloads. You see the artifact list disappear and reappear, but the one that the job was supposed to create isn't returned in the list.

**Comment by ebolyen** [comment]
I believe I have fixed that now.

File: qiime_studio/api/v1.py [hunk 1]
```diff
@@ -107,7 +106,8 @@ def api_input_artifacts(plugin_name, workflow_name, input_name):
     workflow = plugin.workflows[workflow_name]
     input_type = workflow.signature.inputs[input_name][0]
     input_artifacts = []
-    artifact_paths = glob.glob(os.path.join(os.getcwd(), '*.qzf'))
+    path = request.args.get('path', os.getcwd())
```
Comment (jakereps) [inline qiime_studio/api/v1.py]: For some reason the path argument isn't being split into key/values, it's all lumped together like

```python
[1] API: ImmutableMultiDict([('path/Users/Develop/Developer/work/q2-types/q2_types/tests/data', '')])
```

So input artifacts aren't loading, as it reverts to just using the server `.cwd()`

File: app/js/actions/jobs.js [hunk 2]
```diff
@@ -130,9 +131,10 @@ export const linkInputArtifact = (input, artifacts) => ({
 
 export const fetchInputArtifacts = (workflow) => {
     return (dispatch, getState) => {
-        const { connection: { uri, availableApis } } = getState();
+        const { connection: { uri, availableApis }, currentDirectory } = getState();
+        const path = encodeURIComponent(currentDirectory);
         workflow.inputArtifacts.map(input => (
-            fetch(`http://${uri.split('/')[0]}${availableApis[0]}${input.uri}`, {
+            fetch(`http://${uri.split('/')[0]}${availableApis[0]}${input.uri}?path${path}`, {
```
Comment (ebolyen) [inline app/js/actions/jobs.js]: missing an equals

File: qiime_studio/api/v1.py (hunk 1 above)
Comment (ebolyen) [inline qiime_studio/api/v1.py]: fixed

**Pull request merged**